src/Plugin/Field/FieldWidget/InlineEntityFormSingle.php

```diff
@@ -1,151 +0,0 @@
-<?php
-
-/**
- * @file
- * Contains \Drupal\inline_entity_form\Plugin\Field\FieldWidget\InlineEntityFormSingle.
- */
-
-namespace Drupal\inline_entity_form\Plugin\Field\FieldWidget;
-
-use Drupal\Core\Field\FieldItemListInterface;
-use Drupal\Core\Field\WidgetBase;
-use Drupal\Core\Form\FormStateInterface;
-
-/**
- * Single value widget.
- *
- * @FieldWidget(
- *   id = "inline_entity_form_single",
- *   label = @Translation("Inline entity form - Single value"),
- *   field_types = {
- *     "entity_reference"
- *   },
- *   settings = {
- *     "allow_existing" = FALSE,
- *     "match_operator" = "CONTAINS",
- *     "delete_references" = FALSE,
- *     "override_labels" = FALSE,
- *     "label_singular" = "",
- *     "label_plural" = ""
- *   },
- *   multiple_values = false
- * )
- */
-class InlineEntityFormSingle extends WidgetBase {
```

Comment:
Hm.... are we sure we want to remove single widget. This is essentially removing part of functionality that currently exists in D7. I understand multiple widget can handle single value fields too, but UX isn't the same.